### ISSUE TITLE
kiwi generator incompatibility

### ISSUE THREAD

**Issue by jkyon** [vanlooverenkoen/kiwi#15]
Hello

thnak you for this amazing DI, This provide me a lot of functionality.
Now I update some packages to be compatible with mi unit tests. However, kiwi generator is not compatible with test package from sdk.

the error si the following

because kiwi_generator >=0.3.0 depends on analyzer >=0.32.4 <0.35.0 and paymobile_app depends on test ^1.3.0, flutter_test from sdk is incompatible with kiwi_generator >=0.3.0.

Please could you update the packages to fix this compatibility issue?

Thank you in advance

**Comment by amondnet** [comment]
I have the same problem. 
I ended up creating my own fork that:

analyzer: '>= 0.32.4 <0.36.0'

```yaml
dev_dependencies:
  kiwi_generator:
    git:
      url: https://github.com/amondnet/kiwi.git
      path: kiwi_generator
```

**Comment by OttoChamo** [comment]
I followed the flutter example and inside the pubspec.yaml he didn't place the version numbers, if you try it this way it will work .

**Comment by vanlooverenkoen** [comment]
Fixed in kiwi 0.3.0 & kiwi_generator 0.5.0
